.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,105 @@
+name: Feature Request
+description: Suggest an idea you have in mind for this project
+title: "[💡 Feature Request]: "
+labels: ["✨ enhancement"]
+assignees:
+  - Melvin-Abraham
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest an idea for this project!
+
+        🔍 Before you proceed, make sure to search [existing issues](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues) to avoid creating duplicates.
+        💡 Limit this issue to not more than **one feature request** for clarity and discoverability.
+        ✨ Keep the issue concise and clear as much as possible.
+
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: What's the Problem?
+      description: If your feature request is related to a problem, describe it clearly.
+      placeholder: "Eg: I'm always frustrated when [...]"
+
+  - type: textarea
+    id: feature-idea
+    attributes:
+      label: What's your solution or idea?
+      description: "A clear and concise description of what you want to happen"
+      placeholder: "Eg: I would like the assistant to ..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: feature-alternatives
+    attributes:
+      label: Alternatives
+      description: "Describe alternatives you've considered"
+      placeholder: "Eg: I have considered ... as an alternative to this issue"
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ---
+
+        Please include a list of what the feature should and shouldn't do by filling in the table below.
+
+        'Must' implies that the feature should not ship without this capability.
+        'Should' is something we should push hard for, but is not absolutely required to ship.
+        'Could' is a nice-to-have; a good stretch goal that isn't painful if we don't achieve it.
+        'Won't' is a clear statement that the proposal/feature will intentionally not have that capability.
+
+  - type: textarea
+    id: feature-priorities
+    attributes:
+      label: Priorities
+      description: describe all the elements of your idea and how important they are.
+      value: |
+        | Capability | Priority |
+        | :---------- | :------- |
+        | This proposal will allow developers to accomplish W | Must |
+        | This proposal will allow end users to accomplish X | Should |
+        | This proposal will allow developers to accomplish Y | Could |
+        | This proposal will allow end users to accomplish Z | Won't |
```

Comment:
Do we really need this? Opinions?

Reply:
@Melvin-Abraham It is not really necessary, if you ask me. Since I got the basic code from [files](https://github.com/files-community/files) it is from there.

Reply:
Maybe you can remove them

Reply:
They are from the issue form templates that was in GitHub Docs.

Reply:
Cool. I'll remove it then...